packages/listbox/lion-optgroup.js

```diff
@@ -0,0 +1,3 @@
+import { LionOptgroup } from './src/option/LionOptgroup.js';
+
+customElements.define('lion-optgroup', LionOptgroup);
```

Comment:
I would prefer `lion-option-group` to be more coherent with `lion-option`. Having 4 fewer chars is not worth the loss of coherence imo

Reply:
Ahhh so the native element is called optgroup, better to stay aligned with that then :see_no_evil: